index.js

```diff
@@ -1,33 +1,36 @@
 // Promiseの基本パターン
-function countdown (seconds) {
-  return new Promise((resolve, reject) => {
-    const timeoutIds = []
-    for (let i = seconds; i >= 0; i--) {
+function countdown(seconds) {
+  return new Promise((onFulfilled, onRejected) => {
```

Comment:
ERROR: Promise constructor parameters must be named resolve, reject (promise/param-names)